comps/comps-foreman-rhel7.xml

```diff
@@ -202,6 +202,7 @@
       <packagereq type="default">tfm-rubygem-fog-xenserver</packagereq>
       <packagereq type="default">tfm-rubygem-fog-xml</packagereq>
       <packagereq type="default">tfm-rubygem-font-awesome-sass</packagereq>
+      <packagereq type="default">tfm-rubygem-foreman_rh_cloud</packagereq>
```

Comment:
This will need to be in the katello comps file similar to how the inventory upload was